com.unity.formats.fbx/Documentation~/TableOfContents.md

```diff
@@ -1,9 +1,20 @@
 * [FBX Exporter overview](index.md)
-* [Exporting FBX files](exporting.md)
-* [Using the FBX Recorder](recorder.md)
+* [Export GameObjects to FBX](export-gameobjects.md)
+* [Export a Timeline clip to FBX](export-timeline-clip.md)
+* [Record FBX animations in Play mode](export-record-in-play-mode.md)
+  * [Record FBX animations via the Recorder window](export-record-in-play-mode-recorder-window.md)
```

Comment:
this is just a suggestion, but could remove `Record FBX animations` from the subsections as it is already in the main section